WifiManager/impl/WifiManagerSignalThreshold.cpp

```diff
@@ -119,7 +119,7 @@ void WifiManagerSignalThreshold::setSignalThresholdChangeEnabled(bool enabled, i
     {
         int64_t number = std::stoi(response["state"].String());
         state = (WifiState) number;
-        LOGINFO("wifi state = %d", state);
+        LOGINFO("wifi state = %d", (int)state);
```

Comment:
Please do not cast int64_t to int.

Reply:
The "state" is an enum value, not int64_t, so the int typecast will remove warning.